AdminPortal/package.json

```diff
@@ -14,11 +14,15 @@
     "@types/node": "18.11.18",
     "@types/react": "18.0.27",
     "@types/react-dom": "18.0.10",
+    "bootstrap": "^5.2.3",
     "eslint": "8.32.0",
     "eslint-config-next": "13.1.4",
-    "next": "13.1.4",
+    "next": "^13.1.4",
     "react": "18.2.0",
+    "react-bootstrap": "^2.7.1",
     "react-dom": "18.2.0",
+    "react-router-dom": "^6.11.1",
+    "react-select": "^5.7.0",
```

Comment:
Also, I'm unsure why all these dependencies are being changed here. Do we need them? Probably also want to merge main into this branch so it's up to date.

Reply:
have fixed this, merged main and accepted incoming changes for conflicts in package.json